addon/services/launch-darkly-client-remote.js

```diff
@@ -38,14 +39,6 @@ export default Service.extend(Evented, {
       return RSVP.resolve();
     }
 
-    if (!window.LDClient) {
-      warn('Launch Darkly JS client not found. Defaulting all feature flags to "false"', false, { id: 'ember-launch-darkly.client-not-found' });
-
-      this.set('_client', NullClient);
```

Comment:
I believe this means we can remove the `NullClient`

Reply:
Wouldn't we still need it for the case where there is no `clientSideId`?